tools/pontoon/workflow.md

```diff
@@ -0,0 +1,41 @@
+# Example workflow
+
+## AB workflow
+
+This workflow utilizes the full power of Pontoon’s online translation features. It assumes that at least two translators are available for the project; however, this workflow can be completed by a single translator in an **AA workflow** performing each phase or by multiple translators in an **ABC<sub>n</sub> workflow** repeating Phase 3.
+
+### Phase 1: translation
+
+#### Translator A
+
+1. Log in to Pontoon.
+2. Browse to the [project](teams_projects.md).
+3. Using [filters](search_filters.md) select `Missing` and `Fuzzy` to display only the strings currently missing a translation.
```

Comment:
There's also the 'Untranslated' filter which is a shortcut for this, but also includes strings which already have suggestions (which I think is OK for this use case?).